content/darkroom/masking-and-blending/blend-modes.md

```diff
@@ -5,44 +5,159 @@ weight: 20
+
+Lab a-channel
+: Mix the Lab "a" color channel from the input and output images, while taking the other channels unaltered from the input image.
+
+Lab b-channel
+: Mix the Lab "b" color channel from the input and output images, while taking the other channels unaltered from the input image.
 
 Lab color
-: Only available with modules that work in the Lab color space, this blend mode mixes Lab color channels a and b from the input and output images, while lightness is taken unaltered from the input image. In contrast to “color” this blend mode does not involve any color space conversion and does not clamp any data. In some cases this blend mode is less prone to artifacts than “color”.
+: Mix the Lab color channels (a and b) from the input and output images, while taking the lightness unaltered from the input image. In contrast to “color” this blend mode does not involve any color space conversion and does not clamp any data. In some cases this blend mode is less prone to artifacts than “color”.
+
+## RGB channels
+
+The following are available when blending in RGB color spaces only.
+
+RGB red channel
+: Mix the "red" channel from the input and output images, while taking the other channels unaltered from the input image.
+
+RGB green channel
+: Mix the "green" channel from the input and output images, while taking the other channels unaltered from the input image.
+
+RGB blue channel
+: Mix the "blue" channel from the input and output images, while taking the other channels unaltered from the input image.
+
+# HSV channels
+
+The following are available when blending in RGB color spaces only.
 
 HSV lightness
-: Only available with modules that work in the RGB color space, this blend mode mixes lightness from the input and output images, while color is taken only from the input image. In contrast to “lightness” this blend mode does not involve clamping.
+: Mix the lightness from the input and output images, while taking color unaltered from the input image. In contrast to “lightness” this blend mode does not involve clamping.
 
 HSV color
-: Only available with modules that work in the RGB color space, this blend mode mixes color from the input and output images, while lightness is taken only from the input image. In contrast to “color” this blend mode does not involve clamping.
+: Mix the color from the input and output images, while taking lightness unaltered from the input image. In contrast to “color” this blend mode does not involve clamping.
+
+# others
+
+lightness
+: Mix lightness from the input and output images, while taking color (chroma and hue) unaltered from the input image.
+
+chroma
+: Mix chroma (saturation) from the input and output images, while taking lightness and hue unaltered from the input image.
```

Comment:
it's actually chromaticity

Reply:
ok then how does this differ from "color"?

Reply:
"color" uses H and S from HSL (so, expects display-referred RGB). Chroma uses RGB ratios, divided by an euclidean norm, like filmic, color calibration etc. and doesn't care about bounds.

Reply:
I've made some changes from your suggestions but I'm still not 100% certain what you wanted me to write.